### PR TITLE
fix 'raise StopIteration' with Python 3.7 on Fedora 29 rawhide

### PR DESCRIPTION
**Pull request by martinkg**
fix for [issue 26](https://github.com/tiliado/diorite/issues/26)

### CODE REVIEW

File: testgen.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
```
Comment (fenryxo) [inline testgen.py]: Drop this.

File: testgen.py [hunk 2]
```diff
@@ -92,16 +92,16 @@ def __init__(self, name, access, parent=None, rtype=None, params=None, throws=No
...
 	def __repr__(self):
 		buf = ["<Method %s -> %s" % (self.name, self.rtype)]
 		if self.access:
 			buf.append(", %s" % self.access)
 		if self.abstract:
 			buf.append(", abstract")
-		if self.async:
-			buf.append(", async")
+		if self.is_async:
+			buf.append(", is_async")
```
Comment (fenryxo) [inline testgen.py]: Never change async in string literals.

File: testgen.py [hunk 3]
```diff
@@ -237,13 +237,13 @@ def parse_namespace(toks):
 anotations = pp.ZeroOrMore(anotation).setParseAction(parse_anotations)("anotations")
 access = pp.Optional(pp.Keyword("protected") | pp.Keyword("public") | pp.Keyword("private") | pp.Keyword("internal"))("access")
 abstract = pp.Optional(pp.Keyword("abstract"))("abstract")
-async = pp.Optional(pp.Keyword("async"))("async")
+is_async = pp.Optional(pp.Keyword("is_async"))("is_async")
```
Comment (fenryxo) [inline testgen.py]: Never change async in string literals.

**Review by jiri-janousek — CHANGES_REQUESTED** [review]
Send changes to testgen.py as a new pull request referencing #27. Also see inline comments.

**Pull request closed without merging**